internal/handler/web/users.go

```diff
@@ -30,38 +30,28 @@ func (h *Handler) GetUsers(ctx *gin.Context, params openapi.GetUsersParams) {
 		setError(ctx, err, "Failed to get roles for user")
 		return
 	}
-
 	query := params.Find
 
 	// retrieve auth + agreements + training info for set of users
 	if isAdmin {
-		h.getUsersAdmin(ctx, query)
+		h.getUsersAdmin(ctx, &query)
 
 	} else if isTreOpsStaff {
-		h.getUsersTreOps(ctx, query)
+		h.getUsersTreOps(ctx, &query)
 
 	} else {
 		ctx.JSON(http.StatusInternalServerError, "Not implemented")
 	}
 }
 
 func (h *Handler) getUsersAdmin(ctx *gin.Context, query *string) {
```

Comment:
this and `getUsersTreOps` can now take `query` as a `string` (rather than `*query`) as it's required